recipes/spydrpick/build.sh

```
@@ -1,16 +1,33 @@
-make VERBOSE=1 SpydrPick
+
+cmake -S .. -B . -DCMAKE_BUILD_TYPE=Release \
+	-DTBB_ROOT="${PREFIX}" -DBOOST_ROOT="${PREFIX}" \
+	-DCMAKE_CXX_COMPILER="${CXX}" -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
+	-DBoost_DEBUG=1 -DBoost_INCLUDEDIR="${PREFIX}/include" \
+	-DBoost_LIBRARYDIR="${PREFIX}/lib" "${CONFIG_ARGS}"
+cmake --build . --target SpydrPick -j "${CPU_COUNT}" -v
+
 install -d ${PREFIX}/bin
 install bin/SpydrPick ${PREFIX}/bin
 popd
```

Shellcheck

[warning] Use 'popd ... || exit' or 'popd ... || return' in case popd fails.

(SC2164)